### PR TITLE
Feature suggestions UI

### PR DESCRIPTION
**Pull request by faloi**
Closes https://github.com/mumuki/mumuki-platform/issues/282
~~Merge after https://github.com/mumuki/mumuki-classroom-api/pull/110~~
Merge after https://github.com/mumuki/mumuki-office/pull/49

Still in progress, but this actually works. Please comment.

**No suggestion selected:**

image

**Third suggestion selected:**

image

### CODE REVIEW

**Comment by faloi** [comment]
I realize the height of the message modal could be an issue, so I came up with this:

mensaje

**Comment by flbulgarelli** [comment]
Nice. However, what happens if you edit the message after selecting a suggestion?

**Comment by faloi** [comment]
> Nice. However, what happens if you edit the message after selecting a suggestion?

Nothing, the suggestion remains selected. How to deal with further editions is something to be discussed.

However, deselecting the suggestion if the user deletes all the text seems reasonable and easy to implement. What do you think?

**Comment by flbulgarelli** [comment]
> However, deselecting the suggestion if the user deletes all the text seems reasonable and easy to implement. What do you think?

Agree.

File: src/views/modals/new-message.jade [hunk 1]
```diff
@@ -12,6 +12,17 @@
     .container-fluid
       .row
         ace-with-markdown(content='message.message.content', header='new_message')
+
+    .suggestions-panel
+      h5 {{ 'suggestions' | translate }}
+      .suggestion-item(ng-repeat='suggestion in suggestions | limitTo: 3')
+        a(href="", ng-bind-html='suggestion.content_html | trustHtml', ng-click="useSuggestion(suggestion)")
+        span(ng_hide="isSelected(suggestion)")
```
Comment (fedescarpa) [inline src/views/modals/new-message.jade]: Does `ng_hide` works? Use `ng-hide`

File: src/views/modals/new-message.jade [hunk 2]
```diff
@@ -12,6 +12,17 @@
...
+          i.fa.fa-fw.fa-clock-o
+          small {{ Humanizer.shortDate(suggestion.updated_at) }}
+        span(ng_show="isSelected(suggestion)")
```
Comment (fedescarpa) [inline src/views/modals/new-message.jade]: same but with `ng-show`

**Comment by fedescarpa** [comment]
@faloi: 
* "Últimos mensajes enviados para este ejercicio:" is visible even if there isn't suggestions.
* Input-bottom layout is not rendering properly when modal is open.

**Pull request merged**